woocommerce/woocommerce
# Include CSS for experimental tree control

### All Submissions:






### Changes proposed in this Pull Request:



The PR includes the CSS for the experimental tree control, so that is renders properly in Storybook.

The calculation of the tree item padding-left was also moved to CSS from JS.

<img width="1028" alt="Screenshot 2023-01-19 at 17 04 26" src="https://user-images.githubusercontent.com/2098816/213581518-07126413-dd7a-4c71-a922-aee7cfd9e7fa.png">






### How to test the changes in this Pull Request:



0. Build storybook (`pnpm --filter=@woocommerce/storybook build-storybook`)
1. Run storybook (`pnpm --filter=@woocommerce/storybook storybook`)
2. Visit `http://localhost:6007/?path=/story/woocommerce-admin-experimental-treecontrol--simple-tree`
3. A basic tree control should be shown.
4. Verify the CSS is included and the tree looks as shown in the screenshot above.



### Other information:




### FOR PR REVIEWER ONLY:
